packages/jest-cli/src/lib/__tests__/isValidPath-test.js

```diff
@@ -10,32 +10,37 @@
 
 'use strict';
 
+/* eslint-disable max-len */
+
+const path = require('path');
 const isValidPath = require('../isValidPath');
 
+const rootDir = path.resolve(path.sep, 'root');
+
 const config = {
-  rootDir: '/root',
-  roots: ['/root/src', '/root/lib'],
+  rootDir,
+  roots: [path.resolve(rootDir, 'src'), path.resolve(rootDir, 'lib')],
 };
 
 it('is valid when it is a file inside roots', () => {
-  expect(isValidPath(config, '/root/src/index.js')).toBe(true);
-  expect(isValidPath(config, '/root/src/components/Link.js')).toBe(true);
-  expect(isValidPath(config, '/root/src/lib/something.js')).toBe(true);
+  expect(isValidPath(config, path.resolve(rootDir, 'src', 'index.js'))).toBe(true);
```

Comment:
You can do `path.resolve('some/path/with/slash')` and `path` will handle it correctly.

```sh-session
$ node -p -e "require('path').win32.resolve('some/path/with/slash')"
\Users\simen\some\path\with\slash
```